velero-plugins/imagestream/s3_test.go

```diff
@@ -0,0 +1,36 @@
+package imagestream
+
+import "testing"
+
+func TestGetBucketRegion(t *testing.T) {
+	type args struct {
+		bucket string
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    string
+		wantErr bool
+	}{
+		{
+		  	name: "tkaovila-aug30-velero-bsl",
+				args: args{
+					bucket: "tkaovila-aug30-velero-bsl",
+				},
+				want: "us-east-1",
+				wantErr: false,
+		},
+	}
+	for _, tt := range tests {
```

Comment:
how the test knows the region here? did not understanding the where the mocking occurs

Reply:
no mocking, no creds necessary.